### ISSUE TITLE
Resetting push and pull requires workaround

### ISSUE THREAD

**Issue by LarsErikDahl** [nordsoftware/react-foundation#13]
### What did you do?
Createt a column with pushOnSmall={4} and pushOnMedium={0}

### What did you expect to happen?
Rendered HTML: 
```HTML
<div class="small-push-4 medium-push-0" ></div>
```
### What actually happened?
Rendered HTML:
```HTML
 <div class="small-push-4" ></div>
```
### What version of this module are you using?
0.6.8

This is the offending line (line 92 in grid.js):
```js
    props.pushOnMedium ? `medium-push-${props.pushOnMedium}` : null,
```

0 evaluates to false and class is not added. This also applies to a bunch of other classes.

I currently add className={"medium-push-0"} to my column as a workaround, but I would prefer a more uniform option.

**Comment by Jalle19** [comment]
You'll need to escape the HTML somehow, Github has filtered it away.

**Comment by LarsErikDahl** [comment]
Yes, Jalle19, I didn't see that it disappeared.

**Comment by Jalle19** [comment]
Thanks. Since you know where the fault lies, could you perhaps make a PR?

**Comment by crisu83** [comment]
I'll take a look at this when I have time.

**Comment by crisu83** [comment]
This issue should be fixed in version `0.7.0` because now we check against `undefined`, instead of a falsy value.
